tools/jdbc/src/main/java/org/duckdb/DuckDBArrowResult.java

```diff
@@ -0,0 +1,66 @@
+package org.duckdb;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@SuppressWarnings("rawtypes")
+public class DuckDBArrowResult {
+
```

Comment:
Looking at this again, this should probably just implement the iterable/iterator interfaces to ease use and improve interoperability with other java libraries and code